pyproject.toml

```diff
@@ -0,0 +1,133 @@
+[project]
+name = "lemming"
+version = "0.0.2"
+license = {file = "LICENSE"}
+authors = [
+    { name = "Jung koo Kang", email = "jungkoo.kang@ibm.com" },
+    { name = "Tathagata Chakraborti", email = "tchakra2@ibm.com" },
+]
+description = "Human in the loop plan selection"
+readme = "README.md"
+requires-python = ">=3.10"
+
+keywords = [
+  "automated planning",
+  "natural language processing",
+  "process automation",
+  "service composition",
+  "conversational agents",
+  "generative ai",
+  "large language models",
+]
+
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Natural Language :: English",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.10",
+    "Framework :: Pydantic :: 2",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Scientific/Engineering :: Human Machine Interfaces",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "Topic :: Software Development :: User Interfaces"
+]
+
+dependencies = [
+    "fastapi>=0.104.1",
+    "pydantic>=2.5.2",
+    "forbiditerative==1.0.3",
+    "requests==2.30.0",
+    "pydot==1.4.2",
+    "networkx==3.1.0",
+    "python-multipart==0.0.6",
+    "uvicorn[standard] >=0.19.0",
+    "plan4past @ git+https://github.com/whitemech/Plan4Past.git",
+]
+
+[project.optional-dependencies]
+test = ["coverage==7.2.5"]
+dev = [
```

Comment:
separated installation of dev tools to different environment

Reply:
Good!